test/engine-tap/lib/tester.lua

```diff
@@ -10,7 +10,9 @@ local test = tap.test("errno")
 
 local ok, test_run = pcall(require, 'test_run')
 test_run = ok and test_run.new() or nil
-local engine = test_run ~= nil and test_run:get_cfg('engine') or 'memtx'
+local engine = test_run ~= nil and test_run:get_cfg('engine') or
+               os.getenv('TEST_ENGINE')
+engine = engine:gsub('"', '')
```

Comment:
Why do we need this line? Looks like CMake misconfiguration.

Reply:
It is not needed actually, removed:

```diff
--- a/test/engine-tap/lib/tester.lua
+++ b/test/engine-tap/lib/tester.lua
@@ -12,7 +12,6 @@ local ok, test_run = pcall(require, 'test_run')
 test_run = ok and test_run.new() or nil
 local engine = test_run ~= nil and test_run:get_cfg('engine') or
                os.getenv('TEST_ENGINE')
-engine = engine:gsub('"', '')
 
 function test.engine(self)
     return engine
```